src/types/WalletConnectService.ts

```diff
@@ -7,7 +7,7 @@ import type { GasPrice } from './GasPriceType';
 
 export type WalletConnectClientType = WalletConnectClient;
 
-type WCSPendingMethod = '' | 'sendMessage' | 'signJWT' | 'signHexMessage';
+type WCSPendingMethod = '' | 'sendMessage' | 'signJWT' | 'signHexMessage' | 'sendWalletAction';
```

Comment:
I think we should have a pending method named as the exact method, in this case switchToGroup